test/torchaudio_unittest/assets/decoder/lexicon.txt

```diff
@@ -0,0 +1,2 @@
+a a |
```

Comment:
Could you add 
1. words that are composed of multiple of tokens?
2. words that share common ancestor tokens?

I think it's a bit easier to read if words are different from tokens.
 
```
foo f o o |
bar b a r |
foobar f o o b a r |
```